api/src/main/java/edu/cornell/mannlib/vitro/webapp/dao/jena/JenaModelUtils.java

```diff
@@ -2,6 +2,9 @@
 
 package edu.cornell.mannlib.vitro.webapp.dao.jena;
 
+import static edu.cornell.mannlib.vitro.webapp.dao.jena.SparqlGraph.sparqlNode;
+import static edu.cornell.mannlib.vitro.webapp.dao.jena.SparqlGraph.sparqlNodeDelete;
```

Comment:
I am wondering if those static method have been already imported whether we should replace all SparqlGraph.sparqlNode and SparqlGraph.sparqlNodeDelete invocation with sparqlNode and sparqlNodeDelete?  (lines 584 and 588)

Reply:
Yes. Replaced.